apps/agent/src/mastra/workflows/github-notify-developer-about-pr-status.ts

```diff
@@ -60,6 +62,69 @@ async function handleWaitingForReview(
   }
 }
 
+async function handleUnconventionalTitleOrDescription(
+  discordUserId: string,
+  prs: PullRequest[],
+) {
+  const readyToCheckPRs = prs.filter((pr: PullRequest) => !pr.isWIP)
+  const titleFormatRegex = /^([a-zA-Z]+)(\([\w-]+(,[\w-]+)*\))?\:\s(.+)$/
+
+  const wrongConventionPRs = await Promise.all(
+    readyToCheckPRs.map(async (pr: PullRequest) => {
+      // use agent to check if PR description is good
+      const prompt = `Based on the following pull request, help me determine if it needs to be improved:
+        ${JSON.stringify({ title: pr.title, body: pr.body }, null, 2)}`
```

Comment:
Can we review all PRs at once? This could reduce delays. I think it’s possible and might save us latency time.